react/src/components/NavBar/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import { NavLink } from 'react-router-dom';
+import { Dropdown, DropdownToggle, DropdownMenu, DropdownItem } from 'reactstrap';
+import 'bootstrap/dist/css/bootstrap.css';
+
+import UserAvatar from '../UserAvatar';
+import routes from '../../constants/routes'
+import wbooksLogoSvg from '../../assets/wbooks_logo.svg';
+import addBookSvg from '../../assets/add_book.svg';
+import notificationsSvg from '../../assets/notifications.svg';
+
+import strings from './strings';
+import './styles.css';
+
+class NavBar extends Component {
+
+  toggle = () => {
+    this.setState(prevState => ({ dropdownOpen: !prevState.dropdownOpen }));
+  }
+  clickLogOut = () => {
+    sessionStorage.clear();
+    window.location.href = routes.LOGIN();
+  };
+  toggle = this.toggle.bind(this);
```

Comment:
This shouldn't be needed, since `toggle` is already attached to the instance, not to the class itself